pkgs/tools/security/certinfo/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, lib, buildGoModule, fetchFromGitHub, libX11, darwin, Cocoa ? null }:
```

```suggestion
{ stdenv, lib, buildGoModule, fetchFromGitHub, libX11, darwin }:
```
Cocoa is accessed through darwin.* below